### ISSUE TITLE
TypeError: 'NoneType' object in home recommended items

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#3807]
From reporter - Tribler 7.1.0 -beta
```python
Traceback (most recent call last):
File "TriblerGUI\widgets\home_recommended_item.py", line 73, in on_download_button_clicked
TypeError: 'NoneType' object has no attribute '__getitem__'
```

**Comment by qstokkink** [comment]
I fixed a lot of these in #3773

**Comment by xoriole** [comment]
Apparently, there are still some left.
